noir-projects/aztec-nr/aztec/src/macros/dispatch/mod.nr

```diff
@@ -0,0 +1,134 @@
+use protocol_types::abis::function_selector::FunctionSelector;
+use crate::context::inputs::public_context_inputs::PublicContextInputs;
+use super::utils::compute_fn_selector;
+
+/// Returns an `fn public_dispatch(...)` function for the given module that's assumed to be an Aztec contract.
+pub comptime fn generate_public_dispatch(m: Module) -> Quoted {
+    let functions = m.functions();
+    let functions = functions.filter(|function: FunctionDefinition| function.has_named_attribute("public"));
+
+    let public_context_inputs = get_type::<PublicContextInputs>();
```

Comment:
I'm removing the inputs in this PR: https://github.com/AztecProtocol/aztec-packages/pull/8770

Could you update this after it's merged?